### ISSUE TITLE
Does not use full width listing tasks

### ISSUE THREAD

**Issue by Sm03leBr00t** [phaazon/toodoux#62]
It would be a great **feature** if toodoux would use full width listing to save new lines and display more. Weirdly it adjusts if you put in a longer task but always breaks the last word so may even be a **bug**.

![image](https://user-images.githubusercontent.com/36639589/99034968-c4098f80-257e-11eb-9533-931cc44d4df3.png)

**Comment by LordMZTE** [comment]
It should use as many columns as your terminal allows without line wrapping so this is probably a bug.

**Comment by phaazon** [comment]
Yeah I noticed that lately too. I’ll have a look, I think I know why it does this.

**Comment by phaazon** [comment]
The problem comes from the fact the algorithm used to detect when we need to cut a sentence and go to the next line always adds an extra character when computing widths of words (so that we can add the `…`). However, for the _last word_, if it fits, we won’t need the `…`.

Two ways of fixing it: add 1 when computing the description_width (but it will make a weird padding at the right side of the right column) or fix the algorithm. A third option that I don’t really like is to use the full length of the terminal by basically padding with the background color (this is an easy change), but I just don’t like it.
